user-guide/integrations/zapier-integration.md

```diff
@@ -0,0 +1,34 @@
+# Creating a Zap in Zapier for New Query Results
+
+You can connect Redash to Zapier and trigger an action every time a new line is added to you query results - for example, a Slack bot that announces every time a new order came in.
+The best way is to have a scheduled query that refreshes frequently so the notification you get is in real-time.
+
+For now you'll need to ask us for an invitation as it's still in beta - you can do that via Intercom (at the bottom right corner of every page in /help or inside the Redash webapp or [Slack](https://redash.slack.com/).
+
+Here's a short example of connecting Redash to Zapier and creating a new Zap:
+1. Open the invitation you got via email to use the beta Redash trigger app (,create an account if needed) and click "Make a Zap" right next to your account area at the top of the page.
+
+2. Choose a trigger app - in this case, Redash (it won't be available to you unless you have received and opened the invitation)
+
+3. Choose a trigger - for now we have only "New Query Results" in there, this means the Zap will get triggered everytime a new line is added to the results set of your query.
```

Comment:
Worth reiterating that Zapier checks for existing results, so they should schedule the query to refresh according to the frequency they defined Zapier to check for results.